spec/unit/filters/filter_form_builder_spec.rb

```diff
@@ -155,6 +155,26 @@ def filter(name, options = {})
     end
   end
 
+  describe "citext attribute" do
```

Comment:
Ok I tried hard creating a tests and it is possible, but everything is a messy hack unless postgres is introduced.